packages/shared/src/components/BookmarkFeedLayout.tsx

```diff
@@ -122,22 +119,13 @@ export default function BookmarkFeedLayout({
         />
       )}
       {showPlusSubscription && (
-        <div className="mb-4 px-4 laptop:hidden">
-          <Button
-            className="w-full"
-            variant={ButtonVariant.Tertiary}
-            onClick={() => {
-              openModal({
-                type: LazyModal.BookmarkFolderSoon,
-                props: {},
-              });
-            }}
-          >
-            <div className="flex items-center justify-center rounded-6 bg-background-subtle">
-              <PlusIcon size={IconSize.Small} />
-            </div>
-            <div className="w-full pl-4 text-left">New folder</div>
-          </Button>
+        <div className="mb-4 laptop:hidden">
+          <BookmarkFolderSection
+            isItemsButton={false}
+            sidebarExpanded
+            shouldShowLabel
+            activePage=""
```

Comment:
Why are these props needed?

Reply:
I'm reusing the sidebar component here in order to not create a new one from scratch 👀

Reply:
Should maybe raise for product if they don't always want this change already.
Mainly asking because now if someone get's enrolled in plus the sidebar changes quite a bit.

(non blocking for your stuff though)